conda-forge.yml

```diff
@@ -3,3 +3,7 @@ azure:
   settings_win:
     pool:
       vmImage: windows-2022
+noarch_platforms:
+  - linux_64
+  - osx_64
+  - win_64
```

Comment:
You should do that only in the feedstock. We should not change it here in the staged-recipes area. The OS noarch is still something new and we haven't figure out a better way to implement it yet.

Reply:
So we do have a better way of doing it 😜